react/components/Image.js

```diff
@@ -4,12 +4,11 @@ import PropTypes from 'prop-types'
 import productSummary from '../productSummary.css'
 
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, description }) {
   return (
-    <div
-      className={`${className} ${productSummary.aspectRatio} w-100`}
-      style={{ backgroundImage: `url(${src})` }}
-    />
+    <div>
```

Comment:
I think this `div` is not necessary and removing this `div` the component would make any sense too.